### PR TITLE
HDInsight: Powershell changes for creating/deleting IaaS clusters

### PR DESCRIPTION
**Pull request by vhamine**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vhamine__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (vhamine). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

**Comment by AzureRT** [comment]
Can one of the admins verify this patch?

**Comment by vhamine** [comment]
The test failure does not appear to be related to my changes (which are isolated to HDInsight cmdlets). Is there any action item on my side before we proceed with the merge.

**Comment by vhamine** [comment]
Can someone with write access please merge these changes?

File: src/ServiceManagement/HDInsight/Commands.HDInsight/Model/DataObjects/AzureHDInsightCluster.cs [hunk 1]
```diff
@@ -188,5 +188,13 @@ public string VersionStatus
         {
             get { return this.cluster.VersionStatus.ToString(); }
         }
+
+        /// <summary>
+        ///     Gets the type of Operating System installed on cluster nodes.
```
Comment (hovsepm) [inline src/ServiceManagement/HDInsight/Commands.HDInsight/Model/DataObjects/AzureHDInsightCluster.cs]: Please fix the spacing in the comments

File: src/ServiceManagement/HDInsight/Commands.HDInsight/Cmdlet/NewAzureHDInsightClusterCmdlet.cs [hunk 2]
```diff
@@ -47,6 +47,49 @@ public NewAzureHDInsightClusterCmdlet()
...
+         * Parameter Sets:
+         * 
+         * ParameterSetClusterByConfigWithSpecificSubscriptionCredentials
+         *      Position 00: Name
+         *      Position 01: Config
+         *      Position 02: Subscription
+         *      Position 03: Certificate
+         *      Position 04: Location
+         *      Position 05: Credential
+         *      Position 06: ???
```
Comment (hovsepm) [inline src/ServiceManagement/HDInsight/Commands.HDInsight/Cmdlet/NewAzureHDInsightClusterCmdlet.cs]: What does "???" mean? Please add comment if this position is skipped intentionally/by mistake.

File: src/ServiceManagement/HDInsight/Commands.HDInsight/Cmdlet/NewAzureHDInsightClusterCmdlet.cs [hunk 3]
```diff
@@ -47,6 +47,49 @@ public NewAzureHDInsightClusterCmdlet()
...
+         * ParameterSetClusterByNameWithSpecificSubscriptionCredentials
+         *      Position 00: Name
+         *      Position 01: Subscription
+         *      Position 02: Certificate
+         *      Position 03: Location
+         *      Position 04: DefaultStorageAccountName
+         *      Position 05: DefaultStorageAccountKey
+         *      Position 06: DefaultStorageContainerName
+         *      Position 07: Credential
+         *      Position 08: ???
```
Comment (hovsepm) [inline src/ServiceManagement/HDInsight/Commands.HDInsight/Cmdlet/NewAzureHDInsightClusterCmdlet.cs]: What does "???" mean? Please add comment if this position is skipped intentionally/by mistake.

**Pull request merged**